Python/pylifecycle.c

```diff
@@ -680,7 +680,6 @@ pyinit_config(_PyRuntimeState *runtime,
     if (_PyStatus_EXCEPTION(status)) {
         return status;
     }
-    config = &tstate->interp->config;
```

Comment:
Hm, this one I'm having trouble understanding how it happened and if it was actually *intended* to have some effect.

Reply:
This change is correct. I wrote the config, but it's no longer needed to update the config variable.